src/API/Site/Controllers/TourController.php

```diff
@@ -148,6 +153,7 @@ protected function get_schema_properties(): array {
 				'type'              => 'string',
 				'validate_callback' => 'rest_validate_request_arg',
 				'required'          => true,
+				'pattern'           => $this->get_tour_id_regex(),
```

Comment:
I can still create a tour with some symbols like `@` in the tour name. It only works if the regex pattern has a begin and ending sign, e.g. `^[a-zA-z0-9-_]+$`.

Reply:
Amazing catch! Thanks

I adjusted the code again.